app/services/tims.rb

```diff
@@ -2,4 +2,26 @@
 # frozen_string_literal: true
 
 class Tims
+
+  def get_disruptions
+    download_and_filter_disruption.map{ |disruption| Disruption.new(*first_divided(disruption["CauseArea"]["DisplayPoint"]["Point"].values)) }
+  end
+
+  def download_and_filter
+    download_and_filter_disruption
+  end
+
+  private
+
+  def download_xml
+    HTTParty.get('https://data.tfl.gov.uk/tfl/syndication/feeds/tims_feed.xml?app_id=&app_key=').parsed_response
+  end
+
+  def download_and_filter_disruption
+    download_xml["Root"]["Disruptions"]["Disruption"]
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.